### PR TITLE
i18n: integration with Angular templates

### PR DESCRIPTION
**Pull request by jirikuncar**
Signed-off-by: Jiri Kuncar jiri.kuncar@cern.ch

### CODE REVIEW

File: invenio_search_ui/bundles.py [hunk 1]
```diff
@@ -39,6 +47,63 @@
...
+        return list(files())
+
+    contents = property(_get_contents, Bundle._set_contents)
+
+
+class AngularGettextFilter(Filter):
+    name = 'angular-gettext'
+
+    def output(self, _in, out, **kwargs):
+        out.write("angular.module('tindThemeTranslations', ['gettext']).run(['gettextCatalog', function (gettextCatalog) {")
```
Comment (andrewm89) [inline invenio_search_ui/bundles.py]: @jirikuncar is there are way to get the module name?  We can then use this to replace 'tindTheme'.  It should then be easier to do the import of the translations in the angular app based on the module name + 'Translations'

File: invenio_search_ui/bundles.py [hunk 2]
```diff
@@ -39,6 +47,63 @@
...
+        out.write(_in.read())
+        out.write("}]);")
+
+    def input(self, _in, out, **kwargs):
+        import ipdb
+        ipdb.set_trace()
```
Comment (drjova) [inline invenio_search_ui/bundles.py]: I guess the PR is still a WIP :)

Comment (drjova) [inline invenio_search_ui/bundles.py]: cds/modules/deposit/static/json/cds_deposit/forms/project.json

**Comment by jirikuncar** [comment]
Depends on inveniosoftware/invenio-assets#51

**Comment by jirikuncar** [comment]
Since [Invenio-Assets v1.0.0b4](https://pypi.python.org/pypi/invenio-assets/1.0.0b4) is out we have green light on Travis. More reliable language code detection will come with https://github.com/inveniosoftware/invenio-assets/pull/54.

Please note that in order to fully plug translations, there is a work needed on language selector in https://github.com/inveniosoftware/invenio-i18n/issues/34 probably by setting a cookie with selected language that will be shared by frontend and backend.

**Comment by lnielsen** [comment]
LGTM

**Comment by jirikuncar** [comment]
@lnielsen @drjova PR is rebased.

**Comment by jirikuncar** [comment]
@lnielsen finally we have a green light 🎉

**Pull request merged**